Example/CollectionViewToolsExample/Cells/ImageCollectionViewCell.swift

```diff
@@ -6,43 +6,41 @@ import UIKit
 
 final class ImageCollectionViewCell: UICollectionViewCell {
 
-    var removeActionHandler: (() -> Void)?
-
-    let imageView: UIImageView = {
-        let imageView = UIImageView()
-        imageView.contentMode = .scaleAspectFit
-        return imageView
-    }()
-
-    private lazy var removeButton: UIButton = {
-        let button = UIButton(type: .system)
-        button.setTitle("X", for: .normal)
-        button.titleLabel?.font = .systemFont(ofSize: 14, weight: .bold)
-        button.backgroundColor = .white
-        button.addTarget(self, action: #selector(removeButtonPressed), for: .touchUpInside)
-        return button
-    }()
+    var removeActionHandler: (() -> Void)? {
+        didSet {
+            imageContentView.removeActionHandler = removeActionHandler
+        }
+    }
+
+    var imageView: UIImageView {
```

Comment:
```suggestion
    var imageView: UIImageView {
```
private?

Reply:
no, `cellItem` for `factory.cellConfigurationHandler` should has access to it

Reply:
`privet` 👋

Reply:
ok